glean_parser/validate_ping.py

```diff
@@ -23,17 +23,18 @@
 SCHEMAS_DIR = ROOT_DIR / "schemas"
 
 
-PING_SCHEMA_DEFAULT_URL = (
+GLEAN_PING_SCHEMA_GIT_HASH = "a56043b"
+GLEAN_PING_SCHEMA_URL = (
     "https://raw.githubusercontent.com/mozilla-services/"
-    "mozilla-pipeline-schemas/dev/schemas/glean/baseline/"
-    "baseline.1.schema.json"
-)
+    "mozilla-pipeline-schemas/%s/schemas/glean/glean/"
```

Comment:
```suggestion
    "mozilla-pipeline-schemas/{}/schemas/glean/glean/"
```

`.format` uses the newer format syntax.  The `%s` only works with the deprecated `%` operator.